<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>551</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="settingsTabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>541</width>
     <height>451</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="organism_tab">
    <attribute name="title">
     <string>Organisms and simulation</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout">
     <item row="11" column="1">
      <widget class="QSpinBox" name="s_unresolvable_c">
       <property name="minimum">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="s_genome_size">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QCheckBox" name="c_beneficial_mut">
       <property name="text">
        <string>Discard deleterious mutations</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Number of characters/genome size</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QSpinBox" name="s_fitness_target"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Taxon number</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Species difference</string>
       </property>
      </widget>
     </item>
     <item row="18" column="0" colspan="2">
      <widget class="QCheckBox" name="c_recalc">
       <property name="text">
        <string>Recalculate multiplication factor when dialogue closes</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QCheckBox" name="c_sansomian">
       <property name="text">
        <string>Sansomian speciation</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QSpinBox" name="s_taxon_number">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="17" column="0" colspan="2">
      <widget class="QCheckBox" name="c_strip_uninformative">
       <property name="text">
        <string>Strip uninformative characters via subsampling</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="s_organism_mutation">
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Organisms</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Simulation (Organisms)</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QSpinBox" name="s_species_difference">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Rate of organism mutation</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Unresolvable cutoff</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Fitness target</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="envirnment_tab">
    <attribute name="title">
     <string>Environment and extinction</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="4" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Masks per environment</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QCheckBox" name="c_random_overwrite">
       <property name="text">
        <string>Random Overwrite</string>
       </property>
      </widget>
     </item>
     <item row="12" column="3">
      <widget class="QSpinBox" name="s_multiple">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QDoubleSpinBox" name="s_environment_mutation">
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Number of environments</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Playing field size</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QSpinBox" name="s_masks_per_environment">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="16" column="1" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="r_identical">
         <property name="text">
          <string>Identical</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="r_independent">
         <property name="text">
          <string>Independent</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="r_start">
         <property name="text">
          <string>Identical at start</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="3">
      <widget class="QSpinBox" name="s_environment_number">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Selection</string>
       </property>
      </widget>
     </item>
     <item row="10" column="3">
      <widget class="QDoubleSpinBox" name="s_selection">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>2.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Rate of environmental mutation</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QSpinBox" name="s_playingfield_size">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Playing field masks:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Number of playing fields</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Environment</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_20">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Simulation (Environment)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
